configure

```diff
@@ -1061,6 +1061,18 @@ if test "$with_curses" = "yes"; then
   done
 fi
 
+# For instrumented runtime
+# (clock_gettime need -lrt for glibc before 2.17)
+if test "$with_instrumented_runtime" = "true"; then
+  for libs in "" "-lrt"; do
+    if sh ./hasgot $libs clock_gettime; then
+      inf "clock_gettime functions found (with libraries '$libs')"
+      instrumented_runtime_libs="${libs}"
+      break
+    fi
```

Comment:
Should this refuse to build the instrumented runtime if clock_gettime is in fact absent on the system?